deploy/base/pipeline.yaml

```diff
@@ -16,4 +15,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: pipeline
-    namespace: test-jvm-namespace
+    namespace: jvm-build-service
```

Comment:
so if everybody starts testing on openshift, the need for defining pipeline SAs goes away, as the openshift pipelines operator does this for you

WDYT @stuartwdouglas @dwalluck @phillip-kruger ?

in any event, this changes seems redundant with https://github.com/redhat-appstudio/jvm-build-service/pull/236/files#diff-1ff731e4f73618e50718e56e5a434022f7664b1f507d7c3579ddc4f8b44f65a4

Reply:
This is really minikube specific. If we still want to support it it should be part of the minikube scripts.

Reply:
make sense @stuartwdouglas 

one follow up question - with minikube, why does the `pipeline` SA in the `test-jvm-namespace` need the permissions in the `pipeline` clusterrole, namely

```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: pipeline
  labels:
    rbac.authorization.k8s.io/aggregate-to-edit: "true"
rules:
  - apiGroups:
      - jvmbuildservice.io
    resources:
      - artifactbuilds
    verbs:
      - create
  - apiGroups:
      - ""
    resources:
      - configmaps
    resourceNames:
      - jvm-build-config
    verbs:
      - get
```

We are not applying that clusterrole in our openshift ci script.  

Is it because the aggregate-to-edit usage in the operator's rbac.yaml has no effect on minikube?

Assuming that is the case, I think moving to the minikube script entails
- creating the pipeline SA in the test ns from the script
- creating the role and role binding from the script as well, where I delete the files from ./deploy/base and use `cat <<EOF | kubectl apply -f - ..... EOF`

Reply:
this update has been pushed in the latest commit for review